### DISCUSSION TITLE
could not connect to database when importing

### DISCUSSION THREAD

**Discussion by mdcarter** [planetscale/discussion#134] [Bug Reports]
I'm trying to import a Digital Ocean database, and I have this error when trying the connection :
`could not connect to database: Error 1193: Unknown system variable '8Q'`

I tried several combinations of SSL verification mode and TLS version, without luck.

Any ideas ?

**Comment by Phanatic** [comment]
Hi @mdcarter,
  Can you please send me an email with the following information? 
 1. Name of the organization
 2. host name of the database you're trying to connect to.

I'm at Phani At PlanetScale Dot Com

**Reply by mdcarter** [reply]
Hello @Phanatic, did you receive my email ? Any more information about this issue ?
